neurodsp/filt/fir.py

```diff
@@ -46,8 +47,12 @@ def filter_signal_fir(sig, fs, pass_type, f_range, n_cycles=3, n_seconds=None, r
         If True, print out the transition and pass bandwidths.
     plot_properties : bool, optional, default: False
         If True, plot the properties of the filter, including frequency response and/or kernel.
+    save_properties : str
+        Path, including file name, to save filter properites to as a json.
```

Comment:
this file has same comments as iir:
- typo on this line
- any vs np.any on line 87